src/app/src/test/kotlin/org/vaccineimpact/api/tests/controllers/coverage/PopulatingCoverageTests.kt

```diff
@@ -74,6 +80,27 @@ class PopulatingCoverageTests : MontaguTests()
         }.isInstanceOf(ValidationException::class.java)
     }
 
+    @Test
+    fun `deserializing coverage throws error on invalid gavi_support`()
+    {
+        val mockContext = mock<ActionContext> {
+            on { getParts(anyOrNull()) } doReturn MultipartDataMap(
+                    "description" to InMemoryRequestData("some description"),
+                    "file" to InMemoryRequestData(invalidGaviSupportCSVDataString)
+            )
+        }
+        val sut = CoverageController(mockContext, mock(), mock())
+        val exceptionCondition =  Condition<Throwable>(
+                Predicate<Throwable> { e: Throwable ->
+                    val ve = e as ValidationException
+                    ve.errors.count() == 1 && ve.errors[0].code == "csv-bad-data-type:1:gavi_support"
```

Comment:
DataTableDeserializer catches and rethrows the exception thrown by the new Deserializer code [here](https://github.com/vimc/montagu-api/blob/master/src/serialization/src/main/kotlin/org/vaccineimpact/api/serialization/DataTableDeserializer.kt#L99), so we expect the exception here to be slightly different from that thrown by the new code.